modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -57,9 +57,11 @@ This section provides the necessary details that enable you to control egress tr
 |443
 |Provides {op-system-first} images.
 
-|`registry.access.redhat.com`
+|`registry.access.redhat.com` ^[1]^
 |443
-|Provides access to the `odo` CLI tool that helps developers build on OpenShift and Kubernetes.
+|Hosts all the container images that are stored on the Red Hat Ecosytem Catalog. Additionally, the registry provides access to the `odo` CLI tool that helps developers build on OpenShift and Kubernetes.
+
+registry.redhat.io now?  https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/configuring_clusters/install-config-configuring-red-hat-registry
```

Comment:
I see a note in the [old 3.11](https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/configuring_clusters/install-config-configuring-red-hat-registry#:~:text=The%20new%20registry%2C%20registry.redhat.io%2C%20requires%20authentication%20for%20access%20to%20images%20and%20hosted%20content%20on%20OpenShift%20Container%20Platform.) docs that states `registry.redhat.io` replaces `registry.access.redhat.com`?

Is this true? 

registry.redhat.io now? If so, do I need to swap terms?